### ISSUE TITLE
NTR: Hepatobiliary scintigraphy: impaired uptake (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8917]
New term request Hepatobiliary scintigraphy: impaired uptake (elthat_100627165908): Cholescintigraphy, slow

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000930
- label: Impaired hepatobiliary scintigraphy tracer uptake
- parents: [Abnormality of hepatobiliary system physiology (HP:0025155)](https://hpo.jax.org/app/browse/term/HP:0025155)
- pmid: PMID:15347715
- def: An abnormal reduction in the amount of radioactive tracer uptake upon hepatobiliary scintigraphy.
- comment: A normal HBS shows rapid clearance of the radio-tracer from the blood pool. Prompt and uniform hepatic uptake is reached within 5 min, functional visualization of the gall bladder is seen as early as 10 min and bowel activity by 30 to 40 minutes. The presence of tracer in the bowel confirm patency of biliary channels and excludes biliary atresia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8917
- synonyms:
